### ISSUE TITLE
No warning for not indented lines in conditions

### ISSUE THREAD

**Issue by tillhoerner** [martin-helmich/typo3-typoscript-lint#101]
Given the TypoScript
```typoscript
[globalString = GP:foo = 1]
foo.bar = 3
[global]
```
and the configuration
```yaml
sniffs:
  - class: Indentation
    parameters:
      indentConditions: true
```
the linter completes without warnings. Although line 2 should be indented with 4 spaces.

**Comment by julianhofmann** [comment]
Seems there's a regression:
Now, in nested blocks empty lines without indention are showing up a warning.

`page {
  20 = TEXT

  30 = TEXT
}`

"3 Expected indent of 2 spaces."

**Comment by martin-helmich** [comment]
> Seems there's a regression: [...]

@julianhofmann, acknowledged. :+1: Please see #115 for more information.
